doc/howtos/building-images.rst

```diff
@@ -92,10 +122,6 @@ ostro-image:
 ostro-image-dev:
     The same as ostro-image, plus build and debugging tools.
```

Comment:
@mythi , do you expect #76 to be merged soon? If so, we should also add that the *-dev packages will be pre-installed in that image.